### ISSUE TITLE
Battery Connector

### ISSUE THREAD

**Issue by sethkaz** [sethkaz/SuperPower#3]
I want to verify that we are picking the right connector for the battery.  I currently have the JST PH series (aka, the Adafruit standard connector) and a set of vias for soldering wires into. 

Is this sufficient? 

Is there another preferred connector?

**Comment by adoconnection** [comment]
Wont XH 2.54 size connector size will be more DIY friendly?

**Comment by sethkaz** [comment]
I would disagree.  I would say that we would either pick the PH series, as those are almost becoming a standard, or using a 0.1"/2.54mm header (commonly called a Dupont connector).  
The XH 2.54 might have the same pitch, but if we populate the board with it, it will be a pain for the end user if they aren't going to use it.

**Comment by adoconnection** [comment]
After a little thinking I agree that widly used battery connector is better
